packages/cli/test/unit/commands/deploy/index.test.ts

```diff
@@ -1163,17 +1160,6 @@ describe('deploy', () => {
 
         const createdDeploymentId = 'dpl_1';
         client.scenario.post(`/v13/deployments`, (req, res) => {
-          if (missingProjectSettings) {
-            res.status(400).json({
-              error: {
-                code: 'missing_project_settings',
-                framework: null,
```

Comment:
Turns out, `framework: null` is actually an incorrect payload compared to what the API would actually send (it should be the full "Other" framework contents).

This bad payload causes the confirmation step to be skipped in the linking flow. That payload is fixed in this PR, hence the two added `await expect` calls below.